EIPS/eip-1679.md

```diff
@@ -28,6 +28,7 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 
 ### Proposed EIPs
 
+- [EIP-131] : Add Precompiled BLAKE2b Contract
```

Comment:
This needs to be a proper link.

Reply:
Added a link to the PR, will update once merged to draft.
There other option is to link to https://github.com/MadeofTin/EIPs/blob/Blake2b/EIPS/eip-1679.md
If there is a preference.

Reply:
Just use the canonical URL, we can ignore that single failing test to make this easier.